src/changelog.ts

```diff
@@ -180,8 +176,11 @@ export default class Changelog {
     await pMap(
       commitInfos,
       async (commitInfo: CommitInfo) => {
-        if (commitInfo.issueNumber) {
-          commitInfo.githubIssue = await this.github.getIssueData(this.config.repo, commitInfo.issueNumber);
+        const issueData = await this.github.getPullRequest(this.config.repo, commitInfo.commitSHA);
```

Comment:
I think the call to list-pull-requests may need a branch filter on it ... otherwise if working on another branch in the repo and PRing to it, this will return the first created PR that included a commit:

branch: next-feature
PR go to next-feature 
PR next-feature to main
... release

Do you expect to see the PRs to next-feature in the CHANGELOG or the PR of next-feature to main?

Reply:
There is a problem if one passes --from --to params and one of them is not first parent

Reply:
I improved the code to handle duplicate PRs in case of custom from...to params